plugins/Lights/HfdLights.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 UBports Foundation.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: Florian Leeber <florian@ubports.com>
+ */
+
+#ifndef UNITY_HFDLIGHTS_H
+#define UNITY_HFDLIGHTS_H
+
+#include <QtCore/QObject>
+#include <QDBusInterface>
+#include <QtGui/QColor>
+#include "Lights.h"
+
+#define HFD_SERVICE_NAME "com.lomiri.hfd"
+#define HFD_SERVICE_PATH "/com/lomiri/hfd"
+#define HFD_SERVICE_INTERFACE "com.lomiri.hfd.Leds"
```

Comment:
Nitpick: I guess this can be moved to the .cpp file, as they're not used outside the class.